uber/tests/models/test_job.py

```diff
@@ -46,7 +46,8 @@ def test_non_volunteer(self, session):
 
     def test_restricted(self, session):
         assert session.assign(session.staff_two.id, session.job_six.id)
-        session.staff_two.trusted = True
+        session.staff_two.trusted_depts = str(c.CONSOLE)
+        # TODO: update
         session.commit()
         assert not session.assign(session.staff_two.id, session.job_six.id)
 
```

Comment:
Honestly, I think that's a misfeature, because I think that 95% of the time or more, if someone checks the trusted box for a department, it's because they want that person to sign up for shifts in that department.  So by not setting that automatically, we're just creating a workflow where admins will accidentally set one thing but not the other, and then tell someone they can see all shifts when really they can't.

The reason why we do those adjustments (like setting someone as a volunteer if someone selects the volunteer ribbon) is precisely because we had tons of admins doing things like that, where they'd e.g. unset the ribbon but not uncheck the staffing box, and then think that meant someone was no longer a volunteer.

While there might be situations in the future where we'd genuinely want someone to be in a department but not able to sign up for shifts in that department or something, I think that added complexity is asking for confusion.  We have dozens and dozens of admins at this point, most of whom don't know the system that well, so it's important for the "what a newbie expects to happen" to be what actually happens as much of the time as possible.

Just my two cents.